modules/recast/navigation_mesh_editor_plugin.cpp

```diff
@@ -103,24 +103,24 @@ void NavigationMeshEditor::_bind_methods() {
 NavigationMeshEditor::NavigationMeshEditor() {
 
 	bake_hbox = memnew(HBoxContainer);
+
 	button_bake = memnew(ToolButton);
-	button_bake->set_text(TTR("Bake!"));
+	bake_hbox->add_child(button_bake);
 	button_bake->set_toggle_mode(true);
-	button_reset = memnew(Button);
-	button_bake->set_tooltip(TTR("Bake the navigation mesh.") + "\n");
+	button_bake->set_text(TTR("Bake Navigation Mesh"));
```

Comment:
Isn't it a bit long for a tool button? The toolbar is not too crowded yet for NavigationMeshInstance but that may change in the future. Maybe `Bake Mesh` and readd the more verbose tooltip?

Reply:
Renamed it to "Bake NavMesh".